### PR TITLE
docs: new redpanda operator section

### PR DESCRIPTION
**Pull request by micheleRP**
This moves all the original pages with info about the Redpanda Operator into a new Reference section. 

Preview: 
https://deploy-preview-691--redpanda-documentation.netlify.app/docs/reference/redpanda-operator/
https://deploy-preview-691--redpanda-documentation.netlify.app/docs/reference/redpanda-operator/kubernetes-qs-local-access/
https://deploy-preview-691--redpanda-documentation.netlify.app/docs/reference/redpanda-operator/kubernetes-qs-minikube/
https://deploy-preview-691--redpanda-documentation.netlify.app/docs/reference/redpanda-operator/kubernetes-qs-cloud/
https://deploy-preview-691--redpanda-documentation.netlify.app/docs/quickstart/kubernetes-qs-dev/
https://deploy-preview-691--redpanda-documentation.netlify.app/docs/deployment/
https://deploy-preview-691--redpanda-documentation.netlify.app/docs/security/

### CODE REVIEW

File: docs/reference/redpanda-operator/index.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <meta name="description" content="Redpanda Operator reference topics" />
+</head>
+
+This section includes information about installing, deploying, and securing Redpanda on Kubernetes using the Redpanda Operator. This is supported for backward compatibility. 
+
+:::important
+To install with the latest recommended Redpanda Helm chart, see the [60-Second Guide for Kubernetes](../../quickstart/kubernetes-qs-dev).
+:::
+
+Installing Redpanda with the Redpanda Operator:
```
Comment (Feediver1) [inline docs/reference/redpanda-operator/index.mdx:16]: How about: Redpanda Operator installation OR Redpanda Operator Installation

File: docs/reference/index.mdx [hunk 2]
```diff
@@ -14,6 +14,6 @@ Here are some reference pages about what else Redpanda has to offer:
 - [Node Configuration Properties](./node-properties)
 - [Node Configuration Template](./node-configuration-sample)
 - [rpk Commands](./rpk/index.mdx)
-- [Operator Custom Resource Definition (CRD)](https://doc.crds.dev/github.com/vectorizedio/redpanda)
+- [Redpanda Operator](./redpanda-operator/index.mdx)
```
Comment (Deflaimun) [inline docs/reference/index.mdx:17]:
```suggestion
- [Redpanda Operator](../redpanda-operator)
```

File: docs/reference/redpanda-operator/index.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- [60-Second Guide for Kubernetes with minikube](../redpanda-operator/kubernetes-qs-minikube)
+- [60-Second Guide for Kubernetes on Cloud](../redpanda-operator/kubernetes-qs-cloud)
+
+Deploying Redpanda with the Redpanda Operator:
+- [Configuring the Kubernetes Operator for Connectivity](../redpanda-operator/kubernetes-connectivity)
+- [Connecting Remotely to Kubernetes](../redpanda-operator/kubernetes-external-connect)
+- [Custom Configuration in Kubernetes](../redpanda-operator/kubernetes-additional-config)
+- [Using Additional Configuration Parameters for the Redpanda Operator](../redpanda-operator/arbitrary-configuration)
+
+Securing Redpanda with the Redpanda Operator:
```
Comment (Feediver1) [inline docs/reference/redpanda-operator/index.mdx:27]: Redpanda Operator security or Security with Redpanda Operator

**Comment by Feediver1** [comment]
Please add H2 headings in the left sidnav menu: 
- 60-Second Guides
- Deployment
- Security
Rest of the topics can be at the same h1 level they are at now.

File: docs/reference/index.mdx (hunk 2 above)
Comment (Deflaimun) [reply docs/reference/index.mdx]: This has been resolved by https://github.com/redpanda-data/documentation/pull/691/commits/440aa6919a514a8afa64f22a10c2d7db7dba833d

File: docs/reference/redpanda-operator/index.mdx (hunk 1 above)
Comment (micheleRP) [reply docs/reference/redpanda-operator/index.mdx]: > How about: Redpanda Operator installation OR Installing Redpanda Operator

Paulo and I switched both the sidebar and index page to use Install, Deploy, Security.

**Review by joejulian — APPROVED** [review]
lgtm

**Review by Feediver1 — APPROVED** [review]
Thanks @micheleRP

**Pull request merged**